config/routes.rb

```diff
@@ -95,6 +95,8 @@
   get "/comments/page/:page" => "comments#index"
   get "/comments" => "comments#index"
 
+  get "/conversations" => "conversations#index", as: :conversations
```

Comment:
If this can be `resource :conversations, only: [:index, :show]`, you don't need to follow the convention of the rest of this file.

Reply:
Done in ab986b0